scripts/konflux/fetch-vuln-feed-data.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+if [[ "$#" < 1 ]]; then
+  echo "Please pass target directory."
+  exit 1
+fi
+
+TARGET_DIR=$1
```

Comment:
It's good to quote substitutions for consistent style and to avoid issues when paths contain spaces or other funny stuff.

```suggestion
TARGET_DIR="$1"
```

Reply:
FYI: I sort-of forked the script in <https://github.com/stackrox/scanner/pull/1387>. Maybe you'd be interested to use that as a base.